NetKAN/PartOverhauls.netkan

```diff
@@ -0,0 +1,29 @@
+{
+    "spec_version"   : 1,
+    "name"           : "Part Overhauls",
+    "abstract"       : "Porkjet's in-progress part overhauls",
+    "identifier"     : "PartOverhauls",
+    "download"       : "http://bengarner.me/files/PartOverhaulsDistribution.zip",
+    "license"        : "CC-BY-NC-3.0",
+    "version"        : "0.1.0",
+	
+	"author"         : "Porkjet",
+    "release_status" : "development",
+    "ksp_version_min": "1.2.0",
+	
+    "resources" : {
+        "homepage"     : "http://forum.kerbalspaceprogram.com/index.php?/topic/147582-update-12-pre-release-is-here/",

```

Comment:
I tested it and it worked fine, but I guess that's what I get for not linting it...  😕